packages/css/src/plugins/__tests__/atomicify-rules.test.tsx

```diff
@@ -42,6 +42,16 @@ describe('atomicify rules', () => {
     expect(result).toMatchInlineSnapshot(`"._q4hxglyw{-ms-user-select:none;user-select:none}"`);
   });
 
+  it('should double up class selector when two nesting selectors are found', () => {
+    const result = transform`
+      && {
+        display: block;
+      }
+    `;
+
+    expect(result).toMatchInlineSnapshot(`"._1e0c1ule._1e0c1ule{display:block}"`);
```

Comment:
How this will end up in buckets 🤔 in case we have something like `._1e0c1ule:hover._1e0c1ule:hover`? It might break below logic 😞:

```ts
if (sheet.charCodeAt(10) === 58) {
  const openBracketIndex = sheet.indexOf('{');
  const name = sheet.slice(11, openBracketIndex);

  return pseudosMap[name] || '';
}
```

Reply:
ahh! yeah thats a problem isn't it. as well as selectors that have multiples

```
._1e0c1ule:focus, ._1e0c1ule:active {
  color: red;
}
```

HMM

Reply:
It will end up in style bucket 😞

Reply:
Can't we split this into two and always make sure things end up like this => `._1e0c1ule:focus{`

```ts
._1e0c1ule:focus {
  color: red;
}

._1e0c1ule:active {
  color: red;
}
```

Reply:
_we could_ but i'd prefer not creating more baked in styles if we can avoid it, we're already going pretty nuts on it

for this perhaps just place it in the "lowest bucket" and it should be OK? so for this instance it would go in the focus bucket.

Reply:
`._1e0c1ule:focus,._1e0c1ule:active {` will end up in style bucket. Let me change the logic and see if I can fix this one up.